.editorconfig

```diff
@@ -13,6 +13,7 @@ insert_final_newline = true
 trim_trailing_whitespace = false
 
 [*.fs]
+fsharp_max_array_or_list_width=80
 fsharp_max_array_or_list_number_of_items=5
```

Comment:
`fsharp_max_array_or_list_number_of_items` does not do anything on its own.
You need to add `fsharp_array_or_list_multiline_formatter=number_of_items` for this to do anything. And please don't, I regress this setting exists in the first place.